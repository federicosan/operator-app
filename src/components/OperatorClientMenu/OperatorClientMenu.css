.OperatorClientMenu {
  display: flex;
  flex-direction: column;
  flex-grow: 1;
  padding: 0;
  margin: 0;
}

.OperatorClient__Menu__item {
  padding: 0;
  margin: 0;
  list-style-type: none;
}

.OperatorClient__Menu__item:hover {
  background: #fafafa;
}

.OperatorClient__selectedFilter {
  display: flex;
  background: #fafafa;
  border: none;
  outline: 0;
  height: 100%;
  width: 100%;
  padding: 15px 15px 15px 15px;
  border-left: 3px solid rgba(231, 76, 60, 1.0);
}

.OperatorClient__filter {
  display: flex;
  background: #fdfdfd;
  border: none;
  outline: 0;
  padding: 15px 15px 15px 15px;
  height: 100%;
  width: 100%;
  cursor: pointer;
  border-left: 3px solid #fdfdfd;
}

.OperatorClient__filter:hover {
  background: #fafafa;
  border-left: 3px solid rgba(231, 76, 60, 1.0);
}
