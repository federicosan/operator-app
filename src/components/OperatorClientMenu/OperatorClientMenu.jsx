import React, { Component } from 'react';
import { connect } from 'react-redux';

// import PanelMenu from '../PanelMenu/PanelMenu.jsx';

export class OperatorClientMenuComponent extends Component {
  constructor (props) {
    super(props);

    this.state = {};
  }

  render () {
    return (
      <div className="row">
        <ul className="menu" />
      </div>
    );
  }
}

const mapStateToProps = state => ({

});

const mapDispatchToProps = dispatch => ({

});

const OperatorClientMenu = connect(
  mapStateToProps,
  mapDispatchToProps,
)(OperatorClientMenuComponent);

export default OperatorClientMenu;
