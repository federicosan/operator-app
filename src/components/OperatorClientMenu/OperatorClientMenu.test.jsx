import React from 'react';

import { shallow } from 'enzyme';

import OperatorClientMenu from './OperatorClientMenu.jsx';

const store = {
  subscribe: jest.fn(),
  dispatch: jest.fn(),
  getState: jest.fn(() => ({ })),
};

describe('OperatorClientMenu', () => {
  it('matches snapshot', () => {
    const component = shallow(<OperatorClientMenu store={store} />);

    expect(component).toMatchSnapshot();
  });
});
