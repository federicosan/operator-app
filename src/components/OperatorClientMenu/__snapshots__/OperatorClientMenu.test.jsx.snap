exports[`OperatorClientMenu matches snapshot 1`] = `
<OperatorClientMenu
  openChats={0}
  operatorFilter="all"
  setFilter={[Function]}
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "subscribe": [Function],
    }
  } />
`;
